src/server/game/Globals/ConversationDataStore.cpp

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (C) 2008-2017 TrinityCore <http://www.trinitycore.org/>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "ConversationDataStore.h"
+#include "Containers.h"
+#include "Conversation.h"
```

Comment:
This goes against my idea of cleaning up headers - it should be `ConversationDataStore.h` that defines the template structures (or even a separate header), not `Conversation.h` (heavy header to include due to Object.h - pulls all of map & grid)